docs/resources/cloudformation_stack_set.md

```diff
@@ -121,10 +148,43 @@ Required:
 - `key` (String) A string used to identify this tag. You can specify a maximum of 127 characters for a tag key.
 - `value` (String) A string containing the value for this tag. You can specify a maximum of 256 characters for a tag value.
 
+### Advanced StackSet Usage
+To Create a StackSet with Service Managed permissions 
```

Comment:
Please add call out to enable Organizations access before using service-managed permissions